src/openrct2/object/ImageTable.cpp

```diff
@@ -536,9 +536,36 @@ bool ImageTable::ReadJson(IReadObjectContext* context, json_t& root)
             }
             else if (jsonImage.is_object())
             {
-                auto images = ParseImages(context, imageSources, jsonImage);
-                allImages.insert(
-                    allImages.end(), std::make_move_iterator(images.begin()), std::make_move_iterator(images.end()));
+                if (jsonImage.contains("gx"))
+                {
+                    auto xOverride = Json::GetNumber<int16_t>(jsonImage["x"], std::numeric_limits<int16_t>::max());
+                    auto yOverride = Json::GetNumber<int16_t>(jsonImage["y"], std::numeric_limits<int16_t>::max());
+                    const bool hasXOverride = xOverride != std::numeric_limits<int16_t>::max();
+                    const bool hasYOverride = yOverride != std::numeric_limits<int16_t>::max();
```

Comment:
Why not use 0 as the default, even when the user specifies 0 in the json it wouldn't change anything then.

Reply:
Because 0 is actually a valid override value. x/y offsets can be zero or even negative. E.g. if the original `y` offset was -1 and it should be `0`, then you need to be able to specify that.

Reply:
I've misread it a bit, I thought this is used as an offset not a full override, nevermind.